src/Commands.pm

```diff
@@ -2384,6 +2394,55 @@ sub cmdEquip_list {
 	message "================================\n", "info";
 }
 
+sub cmdEquipSwitch {
+	# Equip an item
+	my (undef, $args) = @_;
+	my ($arg1,$arg2) = $args =~ /^(\S+)\s*(.*)/;
+	my $slot;
+	my $item;
+
+	if ($arg1 eq "") {
+		cmdEquipsw_list();
```

Comment:
https://github.com/OpenKore/openkore/blob/3a5b3138f36c60d24f1d292bb4ea6e042c0b0b5b/src/Commands.pm#L2321-L2333

del cmdEquipsw_list(); in cmdEquip ?

Reply:
yep, moved to eqsw